data/examples/README.md

```diff
@@ -217,4 +201,15 @@
     - 45127 water molecules (TIP3P)
 - resolution: all-atom
 - force field: Charmm36m
-- source: generated by Karine Duong with Charmm-Gui
\ No newline at end of file
+- source: generated by Karine Duong with Charmm-Gui
+
+
+`2MAT.gro`
+- molecular inventory:
+    - residues 2 to 263 from E.COLI METHIONINE AMINOPEPTIDASE (PDB: 2MAT)
+    - 200 urea molecules
+    - 200 methanol molecules
+    - 200 ethanol molecules 
+    - 19672 water molecules (TIP3P)
```

Comment:
Are there water molecules or not?

Reply:
No, error with the copy-paste

Reply:
OK but you mentioned [here](https://github.com/pierrepo/grodecoder/issues/39) there was no water.